setup.py

```diff
@@ -209,6 +209,7 @@ def get_local_version(version: "ScmVersion", time_format="%Y%m%d") -> str:
         "tensorboard>=1.14",
         "huggingface_sb3~=2.3",
         "datasets>=2.8.0",
+        "shine",
```

Comment:
What is this for? What's the minimum version required?

Reply:
I've removed this and have instead changed `STABLE_BASELINES3 = "stable-baselines3>=1.7.0,<2.0.0"` to `STABLE_BASELINES3 = "stable-baselines3[extra]>=1.7.0,<2.0.0"`. The bug I got related when running `train_imitation.py` with `sqil` told me to make this change.